test/api.js

```diff
@@ -180,8 +151,9 @@ test('display filename prefixes for failed test stack traces in subdirs', functi
 test('fail-fast mode', function (t) {
 	t.plan(5);
 
-	var api = new Api([path.join(__dirname, 'fixture/fail-fast.js')], {
-		failFast: true
+	var api = new Api({
+		failFast: true,
+		explicitTitles: true
```

Comment:
Actually the test fails without `explicitTitles` set to `true`. The prefixes the test tests for don't show up because it's a single file.